src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -7,43 +7,51 @@
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.InvertType;
 import com.ctre.phoenix.motorcontrol.NeutralMode;
-import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 import static frc.robot.Constants.TankConstants.*;
 
 public class TankSubsystem extends SubsystemBase {
-  private final TalonSRX leftMain;
-  private final TalonSRX leftFollow;
+  private final WPI_TalonSRX leftMain;
+  private final WPI_TalonSRX leftFollow;
 
-  private final TalonSRX rightMain;
-  private final TalonSRX rightFollow;
+  private final WPI_TalonSRX rightMain;
+  private final WPI_TalonSRX rightFollow;
 
-  // motor power output states
+  private final DifferentialDrive differentialDrive;
+
+  // Motor power output states
   private double yScale;
   private double angularScale;
+  private boolean squareInput;
 
   public TankSubsystem() {
-    super();
```

Comment:
Why remove this? We need to call it for the subsystem to be registered.

Reply:
Java should automatically call `super()` for you if you don't explicitly call it with an argument.
> Note: If a constructor does not explicitly invoke a superclass constructor, the Java compiler automatically inserts a call to the no-argument constructor of the superclass. If the super class does not have a no-argument constructor, you will get a compile-time error. Object does have such a constructor, so if Object is the only superclass, there is no problem.
from https://docs.oracle.com/javase/tutorial/java/IandI/super.html

Reply:
:O